build/source/elmfire.f90

```diff
@@ -128,7 +128,7 @@ PROGRAM ELMFIRE
 REWIND(LUINPUT); CALL READ_MONTE_CARLO ; NUM_ENSEMBLE_MEMBERS0 = NUM_ENSEMBLE_MEMBERS
 CLOSE(LUINPUT)
 CALL READ_FUEL_MODEL_TABLE
-IF (USE_BLDG_SPREAD_MODEL .AND. BLDG_SPREAD_MODEL_TYPE .NE. 1) CALL READ_BUILDING_FUEL_MODEL_TABLE
+! IF (USE_BLDG_SPREAD_MODEL .AND. BLDG_SPREAD_MODEL_TYPE .NE. 1) CALL READ_BUILDING_FUEL_MODEL_TABLE

```

Comment:
Rather than commenting this line out please create a dummy building_fuel_models.csv file and commit it to the repo. It could be a single line containing the UCB parameters that you added to elmfire_vars.f90.

Reply:
Done it